src/modules/mavlink/mavlink_stream.h

```diff
@@ -103,6 +103,11 @@ class MavlinkStream
 	virtual bool send(const hrt_abstime t) = 0;
 #endif
 
+	/**
+	 * Function to collect/update data for the streams when update() is called.
```

Comment:
Can you document this a bit more? Like saying that it's called at a high rate, independent from the configured stream rate.